src/main/java/net/mcft/copy/backpacks/container/SlotBackpackWrapper.java

```diff
@@ -0,0 +1,85 @@
+package net.mcft.copy.backpacks.container;
+
+import javax.annotation.Nullable;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.inventory.Container;
+import net.minecraft.inventory.Slot;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+import net.mcft.copy.backpacks.api.IBackpackType;
+
+/** Wraps around the regular player armor slot to
+ *  prevent equipped backpacks from being taken out. */
+public class SlotBackpackWrapper extends Slot {
+	
+	public final Slot base;
+	
+	public SlotBackpackWrapper(Slot base) {
+		super(base.inventory, base.getSlotIndex(), base.xPos, base.yPos);
+		this.base = base;
+	}
+	
+	/** When a backpack is equipped, go through the currently open container, see if
+	 *  any slot contains the player's equipped backpack. If so, replace that slot with
+	 *  a wrapper that prevents the backpack from being unequipped though normal means. */
+	public static void replace(EntityPlayer player, ItemStack backpack) {
+		Container container = player.openContainer;
+		if (container == null) return;
+		for (Slot slot : container.inventorySlots) {
+			if (slot.getStack() != backpack) continue;
```

Comment:
My vision was honestly to check if the slot is a chestplate slot (inventory and position in inventory equal to player's chestplate) and then additionally if it's a backpack, but this might work as well.